### PR TITLE
XKB layouts option: suggest available layouts in the context menu

### LINKED ISSUE #32: Use a dropdown list for keyboard layouts
I have just started using wayfire finding it very great but I had a keyboard problem where it was the US keyboard so i changed the setting in the config manager to `uk` and it didn't work I have just found out that I needed to use the `gb` option. So I think it would be useful to use a drop-down for the keyboard layout where you can press a letter to filter the list.

### PR DESCRIPTION
**Pull request by NamorNiradnug**
Fixes #32 

TODO: the same thing for other XKB-related options

### CODE REVIEW

File: src/wcm.cpp [hunk 1]
```diff
@@ -390,7 +459,18 @@ OptionWidget::OptionWidget(Option *option) : Gtk::Box(Gtk::ORIENTATION_HORIZONTA
...
+                entry = std::make_unique<LayoutsEntry>();
+            } else if (option->name == "xkb_model")
+            {
+                entry = std::make_unique<XkbModelEntry>();
+            } else
+            {
+                entry = std::make_unique<Gtk::Entry>();
+            }
+
             entry->set_text(wf_option->get_value_str());
```
Comment (soreau) [inline src/wcm.cpp:474]: xkb_layout option supports a comma separated list of layouts according to [the wiki](https://github.com/WayfireWM/wayfire/wiki/Configuration#inputxkb_layout). Are you still able to do this with this patch?

Comment (NamorNiradnug) [reply src/wcm.cpp:474]: Yes, this patch only adds suggestions to the context menu of `Gtk::Entry`. It looks like this:
view_2023-03-20_11-07-20

And context menu looks like this
screenshot_2023-03-20-110947

Clicking on one of the layouts appends the layout to the comma separated list (e.g. clicking "au — English (Australian)" appends ", au").

**Comment by NamorNiradnug** [comment]
@soreau Have you finished review? Can this be merged?

**Comment by soreau** [comment]
Makes sense, thanks!

**Pull request merged**